packages/xo-web/src/xo-app/settings/logs/index.js

```diff
@@ -26,6 +26,24 @@ const formatMessage = data =>
     2
   )}\n${JSON.stringify(data.error, null, 2).replace(/\\n/g, '\n')}\n\`\`\``
 
+const formatedLog = log =>
+  `${log.data.method}\n${JSON.stringify(
+    log.data.params,
+    null,
+    2
+  )}\n${JSON.stringify(log.data.error, null, 2).replace(/\\n/g, '\n')}`
+
+const downloadLog = log => {
+  const element = document.createElement('a')
+  // eslint-disable-next-line no-undef
+  const file = new Blob([formatedLog(log)], {
```

Comment:
I don't know if there is a better way to create File, and not trigger `no-undef` from eslint